src/ontology/cl-edit.owl

```diff
@@ -32378,6 +32380,15 @@ SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000101
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002203 obo:CL_0000127))
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001307))
 
+# Class: obo:CL_4042023 (substantia nigra dopaminergic neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32826893") Annotation(oboInOwl:hasDbXref "PMID:35308118") obo:IAO_0000115 obo:CL_4042023 "A midbrain dopaminergic neuron that has its soma located in a substantia nigra. This dopaminergic neuron type is highly metabolically active and it is involved in the regulation of movement, cognition, motivation and reward. Neurodegeneration of this dopaminergic neuronal type causes loss in fine motor control in Parkinson's Disease.")
+AnnotationAssertion(terms:contributor obo:CL_4042023 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042023 "2024-09-16T13:09:29Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:SynonymTypeProperty obo:OMO_0003000) Annotation(oboInOwl:hasDbXref "PMID:32826893") oboInOwl:hasBroadSynonym obo:CL_4042023 "DaN")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35308118") oboInOwl:hasNarrowSynonym obo:CL_4042023 "SNpc dopaminergic neuron")
+AnnotationAssertion(rdfs:label obo:CL_4042023 "substantia nigra dopaminergic neuron"@en)
+
```

Comment:
Classification, location etc still needed

Reply:
Could you be more specific? 
the definition includes:
> A midbrain dopaminergic neuron that **has its soma located in a substantia nigra** . This dopaminergic neuron type is **highly metabolically active** and **it is involved in** the regulation of movement, cognition, motivation and reward. Neurodegeneration of this dopaminergic neuronal type causes loss in fine motor control in Parkinson's Disease.

for classification see the most recent commit.